app/controllers/admin/documentations_controller.rb

```diff
@@ -0,0 +1,33 @@
+class Admin::DocumentationsController < Admin::AdminController
+  include TranslationSaveable
+
+  def new
+    @documentation = Documentation.new(html_identifier: params[:html_identifier])
+    authorize @documentation, :new?
+
+    if params[:caller]
+      controller_action = params[:caller].split('#')
+      @documentation.calling_controller = controller_action[0]
+      @documentation.calling_action = controller_action[1]
+    end
+  end
+
+  def create
+    @documentation = Documentation.new(documentation_params)
+    authorize @documentation
+
+    if @documentation.save
+      # placeholder till other actions are defined
```

Comment:
Any code that we're intending to fix/change later should have comments that start with `TODO:` that way it's easier to find that code later.

Reply:
👍